libnmstate/netapplier.py

```diff
@@ -156,7 +157,15 @@ def _apply_ifaces_state(
                     con_profiles=ifaces_add_configs + ifaces_edit_configs,
                 )
             if verify_change:
-                _verify_change(desired_state)
+                for _ in range(5):
+                    try:
+                        _verify_change(desired_state)
+                        break
+                    except NmstateVerificationError:
+                        time.sleep(1)
+                else:
```

Comment:
I dislike this complex `for else` syntax, especially when ex-BDFL of python dislike it:

https://mail.python.org/pipermail/python-ideas/2009-October/006157.html

Reply:
Fixed. Thanks!